GettingStarted/GettingStarted.md

```diff
@@ -10,10 +10,10 @@ notebook aspect of the LSST science platform. In these notes we provide:
 
 ## Accessing the LSST Science Platform
 The [LSST Science Platform (LSP) Notebook Aspect Documentation](https://nb.lsst.io/) provides an introduction to the NCSA system, including how to gain access and then how to use JupyterLab once you are in.
-Getting in to NCSA takes involves getting an NCSA account, and then figuring out VPN access.
+Getting on to the LSP involves getting an NCSA account, and then figuring out VPN access.
```

Comment:
I would like to start using the term LSST Data Facility instead of NCSA.  It jibes better with the documents we've been producing in the project lately.